### ISSUE TITLE
Regridding model output

### ISSUE THREAD

**Issue by jbusecke** [abby-baskind/seniorthesis#9]
Hi @abby-baskind @gmacgilchrist, 

I am very close to merging [this PR](https://github.com/jbusecke/cmip6_preprocessing/pull/161), which allows to 'automatically' regrid model output from one grid_label to another. 

@abby-baskind Do all the models you are looking at have a 'gr' output for another variable (e.g. `thetao` or `tos`)? Then having them all on a regular grid might be as easy as adding those additional variables and calling:
```python

from cmip6_preprocessing.postprocessing import interpolate_grid_labels

ddict_combined = interpolate_grid_labels(ddict, preferred_grid_label='gr')
```
We could also try to hack together some more code to do this, but it is really not that hard, so I would suggest to implement a regridding step at some point. This will enable you to be sure that what you are showing as a section is truly shown against latitude (and might resolve some of the funkiness in #8 too). Let me know if you have more questions.

**Comment by abby-baskind** [comment]
@jbusecke I'm not really sure how to implement this. Could you explain how this works a bit further?

**Comment by jbusecke** [comment]
Hi @abby-baskind sorry for the delay. Do you think a quick zoom call would be more efficient to clarify this?

**Comment by abby-baskind** [comment]
@jbusecke I'd appreciate that! Maybe Monday?

**Comment by jbusecke** [comment]
Yes. Can you try to schedue a call with [calendly](https://calendly.com/jbusecke)? Have a great weekend!
